engine/pkg/externalresource/broker/interfaces.go

```diff
@@ -46,3 +47,17 @@ type Broker interface {
 
 	Close()
 }
+
+type openStorageOptions struct {
+	needEmpty bool
```

Comment:
Does `needEmpty` mean `cleanBeforeOpen`?

Reply:
Yes, seems `cleanBeforeOpen`  is clearer.